lib/ancestry/class_methods.rb

```diff
@@ -42,23 +42,20 @@ def arrange options = {}
     # Arrange array of nodes into a nested hash of the form
     # {node => children}, where children = {} if the node has no children
     def arrange_nodes(nodes)
-      arranged = ActiveSupport::OrderedHash.new
+      root_id = nil
       min_depth = Float::INFINITY
       index = Hash.new { |h, k| h[k] = ActiveSupport::OrderedHash.new }
 
       nodes.each do |node|
-        children = index[node.id]
-        index[node.parent_id][node] = children
-
+        index[node.parent_id][node] = index[node.id]
```

Comment:
I had this separated intentionally to make the code easier to read...the children variable here matched the comment up above.  I'm ok with removing it, but wasn't sure if you had other reasons other than removing that intermediate variable.